src/ghastoolkit/octokit/octokit.py

```diff
@@ -24,6 +24,9 @@
 
 # logger
 logger = logging.getLogger("ghastoolkit.octokit")
+LOGLEVEL = os.environ.get("LOGLEVEL").upper()
```

Comment:
```suggestion
LOGLEVEL = os.environ.get("LOGLEVEL", "INFO").upper()
```

Maybe set a default if not present?

Reply:
Good shout.